cmake/external/ngraph.cmake

```diff
@@ -54,12 +54,12 @@ if (MSVC)
             PREFIX ngraph
             GIT_REPOSITORY ${ngraph_URL}
             GIT_TAG ${ngraph_TAG}
+            GIT_CONFIG core.autocrlf=input
             PATCH_COMMAND ${NGRAPH_PATCH_DISCARD_COMMAND}
             COMMAND ${CMAKE_COMMAND} -E copy ${PROJECT_SOURCE_DIR}/patches/ngraph/ngraph_onnx.cmake ${ngraph_SRC}/cmake/external_onnx.cmake
             COMMAND git apply --ignore-space-change --ignore-whitespace ${PROJECT_SOURCE_DIR}/patches/ngraph/ngraph_protobuf.patch
-            COMMAND git apply --ignore-space-change --ignore-whitespace ${PROJECT_SOURCE_DIR}/patches/ngraph/ngraph_fix_install_error.patch
-            COMMAND git apply --ignore-space-change --ignore-whitespace ${PROJECT_SOURCE_DIR}/patches/ngraph/ngraph_fix_library_path.patch
             COMMAND git apply --ignore-space-change --ignore-whitespace ${PROJECT_SOURCE_DIR}/patches/ngraph/ngraph_fix_memory.patch
```

Comment:
is the ngraph_fix_memory.patch still needed?
what version of mkldnn is ngraph 0.22 using?

Reply:
Unfortunately it's still needed for 0.22. Amy has tried to backport it but failed with this nGraph release. It will be removed with the next nGraph update which should be 0.25 or 0.26.

0.22 uses a commit instead of a tag to address some issue and the commit itself is https://github.com/intel/mkl-dnn/commit/027de76 (something on top of 0.19 if I understand correctly)